platform/windows/crash_handler_windows_seh.cpp

```diff
@@ -36,7 +36,6 @@
 #include "core/os/os.h"
 #include "core/string/print_string.h"
 #include "core/version.h"
-#include "main/main.h"
```

Comment:
Unrelated, I'd restore to avoid any potential errors from a seemingly unrelated PR